doc/installation.rst

```diff
@@ -76,148 +91,191 @@ On macOS, open the disk image and drag Spyder to your :guilabel:`Applications` f
 .. _releases page: https://github.com/spyder-ide/spyder/releases/latest
 
 
+.. _install-standalone-running:
+
 Running from a standalone install
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To run Spyder when installed standalone, you can simply use your operating system's typical method of launching applications, such as opening it from the Start menu on Windows (or the Taskbar, if you've pinned it there), or from Launchpad, Spotlight or the Applications folder on macOS (or the Dock, if you've added it there).
+To run Spyder when installed standalone, you can simply use your operating system's typical method of launching applications, such as opening it from the :guilabel:`Start` menu on Windows (or the Taskbar, if you've pinned it there), or from Launchpad, Spotlight or the :guilabel:`Applications` folder on macOS (or the Dock, if you've added it there).
 
-On macOS, the first time you open Spyder, you may see a message that it cannot be opened because the developer cannot be verified.
+On macOS, the first time you open Spyder, you may see a message that it cannot be opened because "the developer cannot be verified".
 If so, right-click on the application, select :guilabel:`Open`, then click :guilabel:`Open` in the resulting dialog, and the warning will no longer be shown.
 You can also click :guilabel:`Open Anyway` under :menuselection:`Security & Privacy --> General` in System Preferences.
```

Comment:
```suggestion
```
Our macOS installers are codesigned and notarized, so the user should never encounter this scenario.

Reply:
Thanks; I wasn't sure if this was still a possibility in some scenarios.